uportal-war/src/main/java/org/jasig/portal/rendering/StructureAttributeSource.java

```diff
@@ -30,8 +32,15 @@
  * @version $Revision$
  */
 public class StructureAttributeSource extends StylesheetAttributeSource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(StructureAttributeSource.class);
```

Comment:
This is good.  As a personal preference I suggest in the future using a non-static so you don't have to specify the class because it is an opportunity for an error as code is copied or classes change; e.g. something like
private final Logger LOGGER = LoggerFactory.getLogger(this.getClass());